### PR TITLE
Font cleanup

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: packages/core/style.css [hunk 1]
```diff
@@ -287,13 +265,32 @@ body {
...
+   * fontconfig does not map the name Helvetica Neue to Nimbus Sans, so it's a
+   * better choice than Helvetica here. Arial has a reasonable replacement and
+   * the default sans-serif font is also fine.
+   *
+   * [1] https://nolanlawson.com/2020/05/02/customizing-fonts-in-firefox-on-linux/
+   * [2] https://bugzilla.mozilla.org/show_bug.cgi?id=553580
+   */
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+    "Segoe UI Symbol";
```
Comment (openjck) [inline packages/core/style.css]: Another option is to just copy the <s>antd</s> create-react-app font stack:

```css
font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
  Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue",
  sans-serif;
```

That would solve the same problem and create compatibility with ant, which is probably a good rule of thumb, but it does make slightly different font choices.

**Comment by hamilton** [comment]
@openjck Thanks for taking this on. Your point about antd is a good one. To this end, it feels like maybe we shouldn't actually have _any_ font stack specified, leaving that to the user. We can have one in our templates, but anything beyond that doesn't seem necessary. What do you think?

**Comment by openjck** [comment]
That's a good question. I like that approach. In fact, although some libraries like ant and Material UI do set font stacks on a component-by-component basis, I was mistaken about that particular font stack being from antd. It's actually set by create-react-app.

If it works for you, I'll use that exact font stack in the *app-template* and remove our font stack here. Let me know what you think.

**Comment by hamilton** [comment]
Works great, let's do it. Thanks @openjck!

**Pull request merged**